sysconfig-formula/_states/suse_sysconfig.py

```diff
@@ -35,3 +35,73 @@ def header(name, fillup=None, header_pillar=None):
     pattern=patterns['search'],
     repl=patterns['replace'],
   )
+
+def sysconfig(name, key_values, fillup=None, header_pillar=None, upper=True, append_if_not_found=False):
+  """
+  Manages both the header and the key/value pairs in a sysconfig file.
+  name = sysconfig file name or path to manage
+  upper = whether keys should be converted to upper case
+  All other rguments are passed to the equally named arguments in the suse_sysconfig.header and file.keyvalue functions.
+  """
+  if not name.startswith('/etc/sysconfig/'):
+    name = f'/etc/sysconfig/{name}'
```

Comment:
This will also prefix paths like `/foo/bar`, resulting in `/etc/sysconfig///foo/bar`.

Would checking for `startswith('/')` make more sense, so that only relative paths get prefixed?

Reply:
Interesting point, yes, that sounds sensible.